### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by clayop**
Related to #89

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ For Ubuntu 14.04 LTS users, see this link first:
 
 and then proceed with:
 
+    sudo apt-get update
+    sudo apt-get install autoconf autotools-dev build-essential cmake g++ git libbz2-dev libdb++-dev libdb-dev libicu-dev libreadline-dev libssl-dev libtool openssl python-dev uuid-dev
+    git clone https://github.com/cryptonomex/secp256k1-zkp
```
Comment (maqifrnswa) [inline README.md:16]: If you git submodule update, this should be needed
autogen.sh should be called by cmake, if it doesn't get called we should tell cmake to rebuild it everytime

**Comment by clayop** [comment]
@maqifrnswa Correction is always more than welcome. I just followed instruction from secp256k1-zkp repository, but if there are any mistakes please correct them.

**Comment by maqifrnswa** [comment]
I think you can cut out the lines:

```
cd secp256k1-zkp
./autogen.sh
./configure
make
cd ..
```

as fc's cmake already does it (see https://github.com/cryptonomex/fc/blob/master/CMakeLists.txt#L69)

This will keep it simpler for users.

**Comment by clayop** [comment]
You're right. I will re-pull soon.

**Pull request closed without merging**